locale/ca/docs/index.md

```diff
@@ -0,0 +1,46 @@
+---
+title: Documentació
+layout: docs.hbs
+labels:
+  lts: LTS
+---
+
+# Sobre la documentació
+
+Existeixen tres tipus de documentació a l'abast en aquest lloc:
+
+* Referència de la API
+* Característiques d'ES6
+* Preguntes freqüents
+
+### Referència de l'API
+
+La [Referència de l'API](/api/) proporciona informació detallada sobre una funció ó un objecte en Node.js. Aquesta
+documentació indica quins arguments acepta un mètode, el valor que retorna aquest mètode i quins errors poden estar
+relacionats al mateix. També indica quins mètodes estàn a l'abast per les diferents versions de Node.js
```

Comment:
están, is without accent mark. In fact, I think it could be better if you write here "són"

Reply:
Oki doki :D